packages/api/src/controllers/broadcaster.js

```diff
@@ -59,4 +59,23 @@ app.get('/status', authMiddleware({}), async (req, res, next) => {
   res.json(statuses)
 })
 
+async function discoveryAuthWebhookHandler(req, res) {
+  let responseObj = {}
+  let broadcasters = []
+  try {
+    broadcasters = await req.getBroadcasters(req)
+  } catch (err) {
+    return res.status(400).send({message: err.message})
+  }
+  for (let i = 0; i < broadcasters.length; i++) {
+    if (broadcasters[i].eth.address == req.body.id) {
+      responseObj["priceInfo"] = broadcasters[i].eth?.priceInfo
+      break
+    }
+  }
+  return res.status(200).json(responseObj)
+}
+
+app.get('/hook', discoveryAuthWebhookHandler)
```

Comment:
The `/broadcaster` endpoints are for fetching and querying broadcasters. Should this instead be a part of the `/orchestrator` endpoints which are for fetching orchestrators? Maybe something like `/orchestrator/auth`?

Reply:
Fixed in 2baa3f4